### PR TITLE
Add Profile theme Customization support

### LINKED ISSUE #4834: As a user, I want to be able to customizing my profile 'theme', so I can further represent my brand to the community.
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user, I want to be able to customizing my profile 'theme', so I can further represent my brand to the community.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:

It's been requested by users.
allowing gitcoin profile customizations == people building their identity on top of gitcoin ==  staked identity =  moat for us

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*:
Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
No profile customizations.

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Users can customize their profiles color theme.  
Theme options will be defined in the database.  Each theme will have background color, sub-background color, and font-color. 

Initial options for these themes:
 
default
background: #FFFFFF, sub-back: #EEEEEE, font-color #000000

dark
background: #000000, sub-back: #111111, font-color #FFFFFF

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
see above

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)
number of people returning to the site
number of people returning to the site to manage their profile
number of people who have customized profiles

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

### PR DESCRIPTION
**Pull request by pradeepgangwar**
##### Description

- Added an option under profile tabs to choose profile theme.

##### Refers/Fixes

#4834 

##### Screenshots

Screenshot_2019-12-26  pradeepgangwar Gitcoin

Screenshot_2019-12-29  pradeepgangwar Gitcoin

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
This can be considered as a very initial version of PR. I am open to suggestions. 😄

File: app/dashboard/templates/profiles/tab_theme.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{% load i18n static %}
+{% block settings_content %}
+
+<form method="POST" id="settings" class="col-md-8 m-auto">
+    <div class="form-group">
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tab_theme.html]: indent fix

File: app/dashboard/templates/profiles/tab_theme.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+				</div>
+			</div>
+			<div class="col-md-12 mt-4">
+				<button class='button button--primary' type="submit">{% trans "Save" %}</button>
+			</div>
+		</div>
+	</div>
+    {% csrf_token %}
+</form>
+{% endblock %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tab_theme.html]: could we use 1 tab == 2 spaces

File: app/dashboard/templates/profiles/tabs.html [hunk 3]
```diff
@@ -46,6 +46,13 @@
             </a>
           </li>
         {% endif %}
+        {% if is_my_profile %}
+          <li class="nav-item">
+            <a class="nav-link nav-line  {% if tab == 'theme' %}active{%endif%}" href="{{profile.url}}/theme">
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tabs.html]: `{% endif %}`

File: app/dashboard/templates/profiles/tab_theme.html (hunk 1 above)
Comment (pradeepgangwar) [reply app/dashboard/templates/profiles/tab_theme.html]: @thelostone-mc Fixed those issues. Please review it.

**Comment by pradeepgangwar** [comment]
@thelostone-mc Please review this and let me know if this requires any changes.

**Comment by danlipert** [comment]
@pradeepgangwar looks like this is failing travis tests - can you take a look? thanks!

**Comment by pradeepgangwar** [comment]
@danlipert Pls have a look. All tests pass now. :D

**Comment by thelostone-mc** [comment]
@pradeepgangwar we've decided to step away from this for now as we've got to fix up the css
The dark mode as it stands would need to work throughout the site and use be easy to toggle on and off without having to write custom css rules for every page!

Closing this out for now but thanks for the PR :)

**Pull request closed without merging**